src/containers/PaymentInfo/PaymentInfo.component.js

```diff
@@ -6,31 +6,26 @@ import labeling from 'containers/labeling';
 import PaymentMethod from 'components/PaymentMethod';
 import SectionHeader from 'components/SectionHeader';
 import Transactions from 'components/Transactions';
-import { getPaymentDetails, listCustomerTransactions } from 'api';
+import { listCustomerTransactions } from 'api';
 import { PropTypes } from 'prop-types';
 import UpdatePaymentDetailsPopup from 'components/UpdatePaymentDetailsPopup';
 import { useSelector, useDispatch } from 'react-redux';
 import GracePeriodError from 'components/GracePeriodError';
-import { init } from 'redux/publisherConfigSlice';
+import { init, init as initPublisherConfig } from 'redux/publisherConfigSlice';
```

Comment:
duplicated import, should be: 
`import { init as initPublisherConfig } from 'redux/publisherConfigSlice';`